### DISCUSSION TITLE
Way to disable the "double click to exit full screen"?

### DISCUSSION THREAD

**Discussion by keenankeenankeenan** [sumatrapdfreader/sumatrapdf#4168] [Q&A] [answered]
Is there a way to stop the program from exiting fullscreen when i doubleclick while reading a pdf? When im on the home page it doesnt exit fullscreen when i double click, only when im on a document tab it does so. I couldn't find a way to change this in advanced settings after searching but i might have missed it, is there an option? If there isn't, then fine.

**Comment by kjk** [comment, marked as answer]
In pre-release I improved it to only exit when you double-click outside of text.

However there is no way to disable it. I've added it because there was no way to exit full screen on tablets, without the keyboard.

**Reply by keenankeenankeenan** [reply]
I prefer the improved pre-release way so i'm using that now, thank you
